src/main/java/org/prebid/cache/handlers/CacheHandler.java

```diff
@@ -20,6 +19,8 @@
     ServiceType type;
     static String ID_KEY = "uuid";
 
+    protected String METRIC_TAG_PREFIX;
```

Comment:
This field is not a static constant so the name should follow usual camel-case naming pattern.